setup.py

```diff
@@ -4,9 +4,9 @@
     'develop': {'pytest', 'flake8>=3.9.0', 'black'},
     'local': [
         'pydotplus==2.0.2',
-        'adage==0.10.1',
-        'yadage-schemas==0.10.6',
-        'packtivity==0.14.23',
+        'adage>=0.10.1',
```

Comment:
For reasons yet unknown, _unpinning_ `adage` results in things becoming uninstallable (`pip` taking over 3 hours to find an appropriate solve). I'm not sure why, but I'll follow up on this.

Reply:
```
python -m pip install .[develop,local,kubernetes]
```

works, which seems to imply that the problem is coming from the interaction of `adage` and `reana-client>=0.8.0` (brought in from the `reana` extra).